DEPENDS.md

```diff
@@ -28,6 +28,7 @@ All modules will require the [common](#common) section dependencies.
 - [setproctitle] - Optional: Renaming processes.
 - [Pillow] - Optional: Support for resizing tracker icons.
 - [dbus-python] - Optional: Show item location in filemanager.
+- [ifaddr] - Optional: Verify network interfaces.
```

Comment:
Add at the end of the file `[ifaddr]: https://pypi.org/project/ifaddr/` to make it a link.